test/helpers/ECRecoveryMock.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.11;
+
+
+import '../../contracts/ECRecovery.sol';
+
+
+contract ECRecoveryMock {
+
+  bool public result;
+  address public signer;
+
+  function safeRecover(bytes32 hash, uint8 v, bytes32 r, bytes32 s) {
+    (result, signer) = ECRecovery.safeRecover(hash, v, r, s);
+  }
+
+  function recover(bytes32 hash, bytes sig) {
+    signer = ECRecovery.recover(hash, sig);
+  }
+
+}
```

Comment:
I did it the same way as SafeMath, it uses a mock contract and is a library too. If you want I can run the tests using the lib though.